client/asset/btc/btc.go

```diff
@@ -2439,12 +2446,25 @@ func (btc *ExchangeWallet) convertCoin(coin asset.Coin) (*output, error) {
 // dust) for the change.
 func (btc *ExchangeWallet) sendWithReturn(baseTx *wire.MsgTx, addr btcutil.Address,
 	totalIn, totalOut, feeRate uint64) (*wire.MsgTx, *output, uint64, error) {
```

Comment:
Only the MsgTx and error outputs are used now.  Fine to remove the others I think